pixart.js

```diff
@@ -1 +1,13 @@
+$(document).on("ready", function()
+{
+var setcolor=$("#set-color");
+setcolor.on("click", changeBoxToInput);
 
+var input=$(":input").val();
```

Comment:
try just saving the DOM element in a variable, and then calling methods on it later. if you do this in the event listener, every time the button is clicked, it will grab the value of the brush div.